pkg/policy/policy_controller.go

```diff
@@ -566,6 +570,49 @@ func getTrigger(rd kyverno.ResourceDescription) []*kyverno.ResourceSpec {
 	return specs
 }
 
+func generateTriggers(client client.Interface, rule kyverno.Rule, log logr.Logger) []*unstructured.Unstructured {
+	list := &unstructured.UnstructuredList{}
+
+	kinds := fetchUniqueKinds(rule)
+
+	for _, kind := range kinds {
+		mlist, err := client.ListResource("", kind, "", rule.MatchResources.Selector)
+		if err != nil {
+			log.Error(err, "failed to list matched resource")
+		}
+		list.Items = append(list.Items, mlist.Items...)
+	}
+
+	for _, any := range rule.MatchResources.Any {
```

Comment:
Kinds in `rule.MatchResources.Any` is covered in `fetchUniqueKinds`, why list kinds again?

Reply:
I am using the same `fetchUniqueKinds` kinds here not the any.kinds , ranging over Any to get the any.Selector to list the resources